.pre-commit-hooks.yaml

```diff
@@ -12,3 +12,15 @@
     language: node
     types: [markdown]
     minimum_pre_commit_version: 0.15.0
+-   id: markdownlint-docker
+    name: markdownlint-docker
+    description: "Checks the style of Markdown/Commonmark files, uses projects official docker image."
```

Comment:
I think the part after the comma is not needed - using the same description here seems fine.

Reply:
Changed